src/components/Rating/Rating.visual.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TPAComponentsProvider } from '../TPAComponentsConfig';
+import { VisualContainerElement } from '../../../test/visual/VisualContainerElement';
+import { Rating, Mode, IconSize } from './';
+
+class RatingVisual extends React.Component<any> {
+  static defaultProps = {
+    mobile: false,
+  };
+
+  render() {
+    const { mobile } = this.props;
+
+    return (
+      <TPAComponentsProvider value={{ mobile }}>
```

Comment:
The `TPAComponentProvider` can be removed, since we're not using a `mobile` state for this component.